src/Tabs/InkBar.js

```diff
@@ -16,6 +16,7 @@ function getStyles(props, context) {
       marginTop: -2,
       position: 'relative',
       transition: transitions.easeOut('1s', isRtl ? 'right' : 'left'),
+      zIndex: 1000,
```

Comment:
You know there is no reason for the specific number other than it worked.  Do you have a suggestion as to what it should be relative to the ones you referenced?